### PR TITLE
make the port of SOAP Mock Response test step configurable with property expansion

### PR DESCRIPTION
**Pull request by jpraet**
see https://community.smartbear.com/t5/SoapUI-Open-Source/Scripting-Mock-Response-s-port-number/td-p/34386

### CODE REVIEW

**Comment by olensmar** [comment]
thanks for this - I think we'll need your CLA as described in https://www.soapui.org/developers-corner/contribute-to-soapui.html to move forward!

**Comment by antone-sb** [comment]
@jpraet We'll need your CLA as described in https://www.soapui.org/developers-corner/contribute-to-soapui.html to move forward.

File: soapui/src/main/java/com/eviware/soapui/impl/wsdl/teststeps/WsdlMockResponseTestStep.java [hunk 1]
```diff
@@ -328,6 +328,8 @@ public void prepare(TestCaseRunner testRunner, TestCaseRunContext testRunContext
             assertion.prepare(testRunner, testRunContext);
         }
 
+        mockService.setPort(Integer.parseInt(PropertyExpander.expandProperties(this, mockResponseStepConfig.getPort())));
```
Comment (EkaterinaSorokina) [inline soapui/src/main/java/com/eviware/soapui/impl/wsdl/teststeps/WsdlMockResponseTestStep.java:331]: It looks redundant

**Comment by EkaterinaSorokina** [comment]
@jpraet, please pull the latest changes from next branch. The fix has conflicts with WsdlMockResponseStepStruct class.

File: soapui/src/main/xsd/soapui/soapui.xsd [hunk 2]
```diff
@@ -1118,7 +1118,7 @@
             <xsd:element name="interface" type="xsd:string"></xsd:element>
             <xsd:element name="operation" type="xsd:string"></xsd:element>
             <xsd:element name="path" type="xsd:string"></xsd:element>
-            <xsd:element name="port" type="xsd:int"></xsd:element>
```
Comment (EkaterinaSorokina) [inline soapui/src/main/xsd/soapui/soapui.xsd:1121]: We cannot change data format in the project. It hurts backward compatibility. I'll close the PR.

**Pull request closed without merging**